syz-verifier/verifier.go

```diff
@@ -0,0 +1,119 @@
+// Copyright 2021 syzkaller project authors. All rights reserved.
+// Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+package main
+
+import (
+	"errors"
+	"flag"
+	"fmt"
+	"log"
+	"path/filepath"
+	"strconv"
+	"strings"
+
+	"github.com/google/syzkaller/pkg/mgrconfig"
+	"github.com/google/syzkaller/pkg/osutil"
+	"github.com/google/syzkaller/prog"
+	"github.com/google/syzkaller/vm"
+
+	"github.com/google/syzkaller/sys/targets"
+)
+
+type cfgFlagVals []string
+
+func (cfvs *cfgFlagVals) String() string {
+	return fmt.Sprint(*cfvs)
+}
+
+func (cfvs *cfgFlagVals) Set(value string) error {
+	if len(*cfvs) > 0 {
+		return errors.New("configs flag were already set")
+	}
+	for _, cfv := range strings.Split(value, ",") {
+		*cfvs = append(*cfvs, cfv)
+	}
+	return nil
+}
+
+var (
+	cfvs      cfgFlagVals
+	flagDebug = flag.Bool("debug", false, "dump all VM output to console")
```

Comment:
I know we don't do this in all syzkaller programs (only in some), but please declare all flags in main function. Global variables are still evil, scoping flags in main will allow for better dependencies and testability of code.